example/src/advanced/routeMap.js

```diff
@@ -0,0 +1,59 @@
+import { asRouteMap } from '@kaliber/routing'
+export const routeMap = asRouteMap({
+  root: '',
+  test: { path: '' },
+  language: {
+    path: ':language',
+
+    home: { path: '', data: { title: 'Home' } },
+    articles: {
+      path: { nl: 'artikelen', en: 'articles' },
+      data: { title: { nl: 'Artikelen', en: 'Articles' } },
+
+      list: {
```

Comment:
Goede vraag en dit is een hele bewuste keuze.

Wanneer we een artikel tonen is het niet nodig om alle artikelen op te halen, dat zou wel gebeuren wanneer `list.data` direct in `articles` zou staan. Door hier een losse route van te maken hebben we de vrijheid om te kiezen.

Het patroon wat je voorstelt hebben we wel gebruikt bij `article`: voor elke `tab` moet het artikel aanwezig zijn.

In de FAQ hebben we dit staan:

> - Why is the route map itself not a route?
>   - It would make it impossible to have home as a route that is not the parent of any other routes. This makes some data fetching patterns impossible.